packages/app-staking/src/Account/Unbond.tsx

```diff
@@ -0,0 +1,99 @@
+// Copyright 2017-2019 @polkadot/ui-staking authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { I18nProps } from '@polkadot/ui-app/types';
+
+import BN from 'bn.js';
+import React from 'react';
+import { AccountId } from '@polkadot/types';
+import { Button, InputAddress, InputBalance, Modal, TxButton } from '@polkadot/ui-app';
+
+import translate from '../translate';
+
+type Props = I18nProps & {
+  controllerId?: AccountId | null,
+  isOpen: boolean,
+  onClose: () => void
+};
+
+type State = {
+  maxUnbond?: BN
+};
+
+class Unbond extends React.PureComponent<Props, State> {
+  state: State = {};
+
+  render () {
+    const { controllerId, isOpen, onClose, t } = this.props;
+    const { maxUnbond } = this.state;
+    const canSubmit = !!maxUnbond && maxUnbond.gtn(0);
+
+    if (!isOpen) {
+      return null;
+    }
```

Comment:
See my comments regarding `isOpen` on the index.js. It's cleaner to conditionally render this component in the index.js instead of retuning `null` here.